### ISSUE TITLE
BlockSci Installation not Working; blocksci lib not installed in BlockSci/Notebooks as claimed.

### ISSUE THREAD

**Issue by tejaskhajanchee** [citp/BlockSci#144]
I followed each and every step mentioned [here](https://citp.github.io/BlockSci/readme.html)  followed by [compilation instructions, here](https://citp.github.io/BlockSci/compiling.html), and the local setup instructions [here](https://citp.github.io/BlockSci/setup.html).

But as instructed in the first link, when I go to the BlockSci/Notebooks directory and try to import 'blocksci' in python3 console, the following error occurs:
`Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'blocksci'`

Please Help

**Comment by im2ex** [comment]
Same here

**Comment by im2ex** [comment]
Was some problem with anaconda for me. 
When doing completely without anaconda (disabling in .bashrc), demo notebook works now :-)

**Comment by tejaskhajanchee** [comment]
Hey im2ex!!
Can you please tell me how did you do it? I was not using anaconda in the first place. It still didn't work. I also didn't understand why I am not able to locate a folder named **.bitcoin** in my home folder. Can you please explain why that is happening and  provide me with a solution?

**Comment by RED29** [comment]
jupyter notebook is a visualization. It is better to troubleshoot it within your python. First check which python is the default in your system. I think jupyter notebook is using the default python available. 
Are you using MacOS? If yes, when using macports, "port select list python"

**Comment by tejaskhajanchee** [comment]
@RED29
I am not sure whose comment are you replying to! But in case it's mine, I would reply that I am not using any of the GUI tools. I am working on a Ubuntu 16.04 as recommended in the [documentation](https://citp.github.io/BlockSci/compiling.html). The Ubuntu I am using is a Xenial-64bit Vagrant Box. I don't have MacOS.

**Comment by RED29** [comment]
@tejaskhajanchee my fault. You are right. I deleted my comment to avoid confusion.

**Comment by tejaskhajanchee** [comment]
**_ISSUE SOLVED_**.
The whole installation process was failing at the last command given for the ubuntu setup at [here](https://citp.github.io/BlockSci/compiling.html#ubuntu-16-04). It was because I was using an older version of cmake (3.5.x) I had to upgrade cmake to a new version (3.12.1). After that It worked.

I tested it by importing the blocksci library in python - `import blocksci` or `import blocksci as bs` - and the error that was showing up earlier - `Traceback (most recent call last): File "<stdin>", line 1, in <module> ImportError: No module named 'blocksci'` - is not encountered anymore.
